pipelines/run-generic-pipelines-on-kubeflow-pipelines/README.md

```diff
@@ -0,0 +1,286 @@
+
+### Monitor the pipeline run
+
+Elyra does not provide a monitoring interface for Kubeflow Pipelines. However, it does provide a link to the Kubeflow Central Dashboard's pipeline runs panel.
+
+1. Open the _Run Details_ link. The _runs_ panel is displayed, depicting the in-progress execution graph for the pipeline. Only nodes that are currently executing or have already executed are displayed. Note that the run name is derived from the pipeline name and a timestamp, e.g. `hello-generic-world-0716111722`.
+
+   ![Monitor pipeline run in Kubeflow UI](doc/images/pipeline-graph-run-in-progress.png)
+
+1. Select the first node. A side panel opens, displaying information about the node.
+
+1. Open the _Logs_ tab to access the node's execution log file.
+
+   ![Monitor pipeline run progress](doc/images/review-component-execution-log.png)
+
+   Output that notebooks or scripts produce is captured by Elyra and automatically uploaded to the coud storage bucket you've specified in the runtime configuration.
+
+   If desired, you can visualize results directly in the Kubeflow Pipelines UI. For example, if a notebook trains a classification model, you could visualize its accuracy using a confusion matrix by producing metadata in Kubeflow Pipelines output viewer compatible format.
+
+1. Wait until node processing has completed before continuing.
+
+1. Open the _Visualizations_ tab. For illustrative purposes the first node that runs the `load_data` notebook produces metadata in markdown format, which identifies the data set download location.
+
+   ![Review visualizations](doc/images/review-visualization-in-kfp-gui.png)
+
+   > The code that produces the metadata is located in the notebook's last code cell.
+
+   Refer to [_Visualizing output from your notebooks or Python scripts in the Kubeflow Pipelines UI_](https://elyra.readthedocs.io/en/stable/recipes/visualizing-output-in-the-kfp-ui.html) to learn more about adding visualizations.
+
+1. Wait for the pipeline run to finish.
+
+   ![Wait for pipeline run to finish](doc/images/pipeline-run-completed.png)
+
+### Access the pipeline run outputs
+
+Pipelines that execute on Kubeflow Pipelines store the pipeline run outputs (completed notebooks, script output, and declared output files) in the cloud storage bucket you've configured in the runtime configuration.
+
+1. Open the object storage link and, if required, log in. 
+
+1. Navigate to the bucket that you've specified in the runtime configuration to review the content. Note that the bucket contains a "directory" with the pipeline's Kubeflow run name, e.g. `hello-generic-world-0716111722`.
+
+   ![Review run output artifacts](doc/images/inspect-object-storage-bucket-content.png)
+
+   If pipeline execution completed successfully the bucket contains for each node the following artifacts:
+      - a `tar.gz` archive containing the original notebook or script and, if applicable, its declared file dependencies
```

Comment:
I stumbled a bit over this sentence. Maybe some commas, like `If pipeline execution completed successfully the bucket contains, for each node, the following artifacts:`, would help? Might just be me

Reply:
I agree. Removed "If pipeline execution completed successfully", since we don't state what happens in case there is an error.